Packs/Mimecast/ReleaseNotes/2_5_5.md

```diff
@@ -0,0 +1,6 @@
+#### Integrations
+
+##### Mimecast v2
+
+- Updated the **mimecast-get-group-members** command to support the **all_results** argument, enabling the retrieval of all results.
+- Updated the **mimecast-get-group-members** command to respect the **limit** argument when it exceeds the default limit, retrieving up to the specified number of results instead of the default amount.
```

Comment:
```suggestion
- Updated the ***mimecast-get-group-members*** command to support the *all_results* argument, enabling the retrieval of all results.
- Updated the ***mimecast-get-group-members*** command to respect the *limit* argument when it exceeds the default limit, retrieving up to the specified number of results instead of the default amount.
```